src/systemctl/systemctl.c

```diff
@@ -6225,7 +6225,7 @@ static int switch_root(int argc, char *argv[], void *userdata) {
         if (init) {
                 const char *root_systemd_path = NULL, *root_init_path = NULL;
 
-                root_systemd_path = prefix_roota(root, "/" SYSTEMD_BINARY_PATH);
+                root_systemd_path = prefix_roota(root, "/" ROOTLIBEXECDIR "/systemd");
```

Comment:
That slash here is useless, since `SYSTEMD_BINARY_PATH` or `ROOTLIBEXECDIR` are always absolute.